docs/index.rst

```diff
@@ -23,6 +23,7 @@ Welcome to SOTODLib's documentation!
    preprocess.rst
    det_match.rst
    site_pipeline.rst
+   calibration.rst
```

Comment:
This is currently not being rendered properly. When I try to run I get this error:
```
WARNING: toctree contains reference to document 'calibration' that doesn't have a title: no link will be generated
```

I also it would be helpful if you added some documentation on what the module is doing, how to use it, how to run it in an automated/configuration file sense, etc. in a section of the calibration.rst called wiregrid. See some of the examples/notes/overviews in some of the other modules for examples.